app/routes/events/components/EventDetail/index.js

```diff
@@ -295,26 +288,58 @@ export default class EventDetail extends Component<Props> {
               </Flex>
             </ContentMain>
             <ContentSidebar>
-              <InfoList items={infoItems} />
+              {event.company && (
+                <div className={styles.iconWithText}>
+                  <Icon name="briefcase-outline" className={styles.infoIcon} />
+                  <strong>
+                    <Link to={`/companies/${event.company.id}`}>
+                      {event.company.name}
+                    </Link>
+                  </strong>
+                </div>
+              )}
+              <div className={styles.iconWithText}>
+                <Icon name="time-outline" className={styles.infoIcon} />
+                <strong>
+                  <FromToTime from={event.startTime} to={event.endTime} />
+                </strong>
+              </div>
               {event.isPriced && (
-                <div className={styles.paymentInfo}>
-                  <strong>Dette er et betalt arrangement</strong>
-                  <InfoList items={paidItems} />
+                <div className={styles.iconWithText}>
+                  <Icon name="cash-outline" className={styles.infoIcon} />
+                  <strong>{event.priceMember / 100},-</strong>
                 </div>
               )}
+              <div className={styles.iconWithText}>
+                <Icon name="pin-outline" className={styles.infoIcon} />
+                <strong>{event.location}</strong>
+              </div>
               {event.mazemapPoi && (
-                <MazemapEmbed mazemapPoi={event.mazemapPoi} />
+                <>
+                  <div
+                    className={styles.simulateLink}
+                    onClick={() =>
+                      this.setState({ mapIsOpen: !this.state.mapIsOpen })
+                    }
+                  >
+                    {' '}
+                    Vis kart
```

Comment:
Toggle to "Skjul kart" after it has been clicked (the toggle functionality works, but the text is always "Vis kart" as it is now)